src/widgets/forms/form-view.component.tsx

```diff
@@ -23,8 +30,23 @@ const FormView: React.FC<FormViewProps> = ({
   patientUuid,
   encounterUuid
 }) => {
+  const searchTimeout = 300;
   const { t } = useTranslation();
   const [activeVisit, setActiveVisit] = React.useState<visitItem>();
+  const [searchTerm, setSearchTerm] = React.useState<string>(null);
+  const [allForms, setAllForms] = React.useState<Array<Form>>(forms);
```

Comment:
`allForms` doesn't seem like a very good name, since `forms` is in fact all forms, and `allForms` is a subset of those forms. Maybe `visibleForms`?

Reply:
Update the name to `forms`